deepspeed/runtime/config.py

```diff
@@ -204,11 +204,14 @@ def get_dynamic_loss_scale_args(param_dict):
             FP16_LOSS_SCALE_WINDOW,
             FP16_MIN_LOSS_SCALE,
             FP16_HYSTERESIS,
+            FP16_CONSECUTIVE_HYSTERESIS,
         ]
         if any(arg in list(fp16_dict.keys()) for arg in dynamic_loss_args):
             init_scale = get_scalar_param(fp16_dict, FP16_INITIAL_SCALE_POWER, FP16_INITIAL_SCALE_POWER_DEFAULT)
             scale_window = get_scalar_param(fp16_dict, FP16_LOSS_SCALE_WINDOW, FP16_LOSS_SCALE_WINDOW_DEFAULT)
             delayed_shift = get_scalar_param(fp16_dict, FP16_HYSTERESIS, FP16_HYSTERESIS_DEFAULT)
+            consecutive_hysteresis = get_scalar_param(fp16_dict, FP16_CONSECUTIVE_HYSTERESIS,
+                                                      FP16_CONSECUTIVE_HYSTERESIS_DEFAULT)
             min_loss_scale = get_scalar_param(fp16_dict, FP16_MIN_LOSS_SCALE, FP16_MIN_LOSS_SCALE_DEFAULT)
             loss_scale_args = {
                 INITIAL_LOSS_SCALE: 2**init_scale,
```

Comment:
I don't think `consecutive_hysteresis` is being used currently in the PR. Looking at how the code is used w. ZeRO stage 1 and 2 at least, I think we need to pass `consecutive_hysteresis` to the `loss_scale_args`. I could be wrong though, it's been a bit since I've looked at this part of the code.

Reply:
Ah that was a big oversight on my part. It's kind of hard to test this feature so this slipped through the cracks.

I think it's resolved in https://github.com/microsoft/DeepSpeed/pull/3553/commits/156ef6c120528347ac2b096f816c2dc6e54cd734. Please double check me.